README.md

```diff
@@ -1,6 +1,7 @@
 # Nipoppy
 
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.8084759.svg)](https://doi.org/10.5281/zenodo.8084759)
+[![PyPI - Version]([https://img.shields.io/pypi/v/nipoppy])](https://pypi.org/project/nipoppy/)
```

Comment:
Seems to be causing the docs build to fail with error:
```
/home/docs/checkouts/readthedocs.org/user_builds/nipoppy/checkouts/326/README.md:4: WARNING: image file not readable: ../../../%5Bhttps:/img.shields.io/pypi/v/nipoppy%5D [image.not_readable]
```

Not sure why, and previously it was fine...

Reply:
I think made an error in the link. It has extras square brackets around the url.